.github/ISSUE_TEMPLATE/g3_bug.yml

```diff
@@ -0,0 +1,132 @@
+name: Report a bug
+description: |
+  You found a bug in Flutter causing your application to crash or
+  throw an exception, a widget is buggy, or something looks wrong.
+labels: 'from: g3'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for using Flutter!
+
+        If you are looking for support, please check out our documentation
+        or consider asking a question on Stack Overflow:
+
+          - https://flutter.dev/
+          - https://api.flutter.dev/
+          - https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+  - type: markdown
+    attributes:
+      value: |
+        Before filling the form fields, please consider the following:
+        - Ensure that you have searched the [existing issues](https://github.com/flutter/flutter/issues)
+        - Read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)
```

Comment:
Maybe note here that the tracker is publicly visible and not to submit any internal information or links to internal resources.